src/server/game/Handlers/MovementHandler.cpp

```diff
@@ -725,3 +725,35 @@ void WorldSession::HandleSummonResponseOpcode(WorldPacket& recvData)
     _player->SetSummonAsSpectator(false);
     _player->SummonIfPossible(agree, summoner_guid);
 }
+
+void WorldSession::HandleMoveTimeSkippedOpcode(WorldPacket& recvData)
+{
+    sLog->outString("WORLD: Received CMSG_MOVE_TIME_SKIPPED");
```

Comment:
Need log debug

Reply:
Winfi it's right, this needs to be only for debug purposes.

Reply:
Example here https://github.com/search?q=org%3Aazerothcore+sLog-%3Edebug&type=Code (need to find the right category/filter)